src/main/java/com/onlinebanking/enums/UserHistoryType.java

```diff
@@ -0,0 +1,38 @@
+package com.onlinebanking.enums;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Class holds User history types relevant to a user of the application.
+ *
+ * @author George on 6/29/2021
+ * @version 1.0
+ * @since 1.0
+ */
+@Getter
+@RequiredArgsConstructor
+public enum UserHistoryType {
+
+    /**
+     * This is when a user is created by the system.
+     */
+    CREATED("Account created"),
+
+    /**
+     * This is when the user verifies email.
+     */
+    VERIFIED("Account verified"),
+
+    /**
+     * This is will be used for password changes.

```

Comment:
Remove the 'is' from this line

Reply:
Don't upset the boss

Reply:
Haha the boss has his eye on me on reviews so I'm trying to do a better job.

Reply:
me too